controller/pb/controller.proto

```diff
@@ -118,6 +131,7 @@ message CommonOptions {
   bool Pull = 4;
   bool ExportPush = 5;
   bool ExportLoad = 6;
+  bool Linked = 7;
```

Comment:
`Linked` is internal. I'm a bit on the fence to have it in the proto def: https://github.com/docker/buildx/blob/4a73abfd645457ee8819855cb1a337ab23f833dc/build/build.go#L93-L94

Reply:
Agreed - but based on the current usage of it, I think we need to. It's used in `BuildWithResultHandler` which will be called from the controller side, while it's generated from targets before we make the controller request.

I think it's only used here? https://github.com/docker/buildx/blob/4a73abfd645457ee8819855cb1a337ab23f833dc/build/build.go#L688-L690

I'm quite sure how else we can do it, though open to suggestions :thinking:

Reply:
Ah right let's keep then :+1: